test/integration/src/test/java/io/pravega/test/integration/BatchClientTest.java

```diff
@@ -119,13 +121,42 @@ public void tearDown() throws Exception {
         zkTestServer.close();
     }
 
+    //region Factory methods that may be overridden by subclasses.
+
+    protected ServiceBuilder createServiceBuilder() {
+        return ServiceBuilder.newInMemoryBuilder(ServiceBuilderConfig.getDefaultConfig());
+    }
+
+    protected ControllerWrapper createControllerWrapper() {
+        return new ControllerWrapper(zkTestServer.getConnectString(),
+                false, true,
```

Comment:
A variable name explaining what those boolean values represents can be useful instead of passing `true` or `false`

Reply:
Since it is a simple argument (and not a complex condition/expression), IDE parameter name hints should be enough, assuming the parameter names are aptly-named - which, luckily is the case here as shown in the screenshot below.  

![image](https://user-images.githubusercontent.com/16592107/57351320-0e86a080-717f-11e9-9efd-da684b4a00c5.png)